e2e/package.json

```diff
@@ -9,7 +9,7 @@
     "test": "cross-env WORKBENCH_ENV=dev jest --detectOpenHandles",
     "test:debug": "cross-env WORKBENCH_ENV=dev HEADLESS=false jest --detectOpenHandles",
     "test-local": "cross-env WORKBENCH_ENV=local jest",
-    "test:ci": "cross-env NODE_ENV=development WORKBENCH_ENV=dev CI=true jest --ci --runInBand --forceExit --reporters=default --reporters=jest-junit",
```

Comment:
Junit XML file still saved after removing `--reporters=default --reporters=jest-junit` flags because it caused CircleCI error
```
Error: Could not resolve a module for a custom reporter.
  Module name: tests/home/home-page.spec.ts
    at /home/circleci/*********/e2e/node_modules/jest-config/build/normalize.js:618:15
    at Array.map (<anonymous>)
    at normalizeReporters (/home/circleci/*********/e2e/node_modules/jest-config/build/normalize.js:600:33)
    at normalize (/home/circleci/*********/e2e/node_modules/jest-config/build/normalize.js:693:5)
    at readConfig (/home/circleci/*********/e2e/node_modules/jest-config/build/index.js:220:68)
    at async readConfigs (/home/circleci/*********/e2e/node_modules/jest-config/build/index.js:401:26)
    at async runCLI (/home/circleci/*********/e2e/node_modules/@jest/core/build/cli/index.js:203:59)
    at async Object.run (/home/circleci/*********/e2e/node_modules/jest-cli/build/cli/index.js:216:37)
error Command failed with exit code 1.
```